### ISSUE TITLE
.NET 6

### ISSUE THREAD

**Issue by WORMrus** [uberhalit/EldenRingFpsUnlockAndMore#73]
Hey,

Thanks for the tool, much appreciated. 

Saw your comments regarding .Net 6 not being included in Windows which caused you to fallback to older .Net Framework versions and use slower code (i.e. not FindPattern_SIMD).

If the concern is that an average used would not be able to "just run" a release from this repo, .Net 6 (probably even 5 or 3.1) allows publishing a self-contained exe. I.e. end-users would not have to install anything. Soo... JFYI

See here: https://docs.microsoft.com/en-us/dotnet/core/deploying/#publish-self-contained

**Comment by uberhalit** [comment]
Hello, unfortunately self-contained apps do not fully support WPF (missing trimming support) yet so you end up with a **huge** number of additional dlls and config files that will be bundled into your exe. In my initial test the output was ~100MiB. That is ridiculous for such a small app.

**Comment by DubbleClick** [comment]
I don't think this is feasible either. Microsoft will not start shipping .NET 6 with Windows in the foreseeable future and plaguing every user with a 150mb executable instead of a 300kb one is not a great idea.

At best we could release two versions (self-contained and framework dependent), but even then the framework dependent version would be significantly larger than one targeting .NET Framework.

**Comment by WORMrus** [comment]
Fair enough, 150 megs for a small tool is indeed quite a bit.

It seems that the only feasible way to create cross-platform GUI apps that don't take so much HDD space is something like Blazor. Blazor apps should be trimmable. 

Depending on how strong the benefits of .Net 6 are in this case (I suppose it is mostly about cross-platform builds here and not so much about the performance), it might be worth considering.

**Comment by DubbleClick** [comment]
Rather pointless for a Windows-only application. The only benefit would be C# 10 and speed, the latter of which is irrelevant.
